wpilibc/athena/src/PIDController.cpp

```diff
@@ -332,6 +334,33 @@ void PIDController::SetOutputRange(double minimumOutput, double maximumOutput) {
 }
 
 /**
+ * Set the maximum error value that will cause the accumulated integral to be
+ * calculated. If the error term is large, it can potentially accumulate to have
+ * a huge effect on the output of the PID calculation. Any time the error is
+ * larger than the value specified in this function, the running integral will
+ * be set to zero causing it to not effect the output until the error is closer
+ * to the setpoint.
+ *
+ * <p>The default maximum error to integrate is positive infinity.
+ *
+ * @param maximumErrorToIntegrate the maximum error where integration will occur
+ */
+void PIDController::SetMaxErrorToIntegrate(double maximumErrorToIntegrate) {
+  m_maxErrorToIntegrate = maximumErrorToIntegrate;
```

Comment:
Users could break this if they pass in a negative number. We should specify this to be a magnitude.